index.html

```diff
@@ -10586,114 +10642,120 @@ <h4 id="Spatialzation-equal-power-panning">
     x = azimuth / 90;
   }
-        
-</pre>
+
+              </pre>
+                </li>
+              </ol>
             </li>
           </ol>
           <p>
             For a <a><code>StereoPannerNode</code></a>, the following algorithm
             MUST be implemented.
           </p>
           <ol>
-            <li>
-              <p>
-                Let <em>pan</em> be the <a>computedValue</a> of the
-                <code>pan</code> <a>AudioParam</a> of this
-                <a><code>StereoPannerNode</code></a>.
-              </p>
-            </li>
-            <li>
-              <p>
-                Clamp <em>pan</em> to [-1, 1].
-              </p>
-              <pre>
-pan = max(-1, pan);
-pan = min(1, pan);
-        
-</pre>
-            </li>
-            <li>
-              <p>
-                Calculate <em>x</em> by normalizing <em>pan</em> value to [0,
-                1]. For mono input:
-              </p>
-              <pre>
-x = (pan + 1) / 2;
-        
-</pre>
-              <p>
-                For stereo input:
-              </p>
-              <pre>
-if (pan &lt;= 0)
-  x = pan + 1;
-else
-  x = pan;
-        
-</pre>
-            </li>
-          </ol>
-          <p>
-            Then following steps are used to achieve equal-power panning:
-          </p>
-          <ol>
-            <li>
-              <p>
-                Left and right gain values are calculated as:
-              </p>
-              <pre>
-gainL = cos(x * Math.PI / 2);
-gainR = sin(x * Math.PI / 2);
-        
-</pre>
-            </li>
-            <li>
-              <p>
-                For mono input, the stereo output is calculated as:
-              </p>
-              <pre>
-outputL = input * gainL;
-outputR = input * gainR;
-        
-</pre>
+            <li>For each sample to be computed by this <a>AudioNode</a>
+              <ol>
+                <li>
+                  <p>
+                    Let <em>pan</em> be the <a>computedValue</a> of the
+                    <code>pan</code> <a>AudioParam</a> of this
+                    <a><code>StereoPannerNode</code></a>.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    Clamp <em>pan</em> to [-1, 1].
+                  </p>
+                  <pre>
+    pan = max(-1, pan);
+    pan = min(1, pan);
+
+              </pre>
+                </li>
+                <li>
+                  <p>
+                    Calculate <em>x</em> by normalizing <em>pan</em> value to
+                    [0, 1]. For mono input:
+                  </p>
+                  <pre>
+    x = (pan + 1) / 2;
+
+              </pre>
+                  <p>
+                    For stereo input:
+                  </p>
+                  <pre>
+    if (pan &lt;= 0)
+      x = pan + 1;
+    else
+      x = pan;
+
+              </pre>
+                </li>
+              </ol>
```

Comment:
We need to spell out the algorithm for each sample instead of for each block, so this change the indentation level so this looks like a big change.

If I go a `git diff -w HEAD^^^` in my terminal, that is, diff against master without whitespace changes, it looks better, but I don't think github supports that.